src/packs/core_macros/Scan__Journal__z0pdvwE0FitMzwRJ.yml

```diff
@@ -60,6 +60,12 @@ command: >-
 
   const updateExisting = true;
 
+  //actorPermissionLevel - This sets the permission level for the parent actor
+  of a token. This must be an integer between -1 and 3 where -1 is "No change",
+  0 is "None", 1 is "Limited", 2 is "Observer", and 3 is "Owner"
+
+  const actorPermissionLevel = -1;
```

Comment:
May want to use the CONST values for readability

Reply:
Should be addressed now! Resolving